benchmark/sirun/appsec-iast/common.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+module.exports = {
+  port: 3231 + parseInt(process.env.CPU_AFFINITY || '0'),
+  reqs: 150
```

Comment:
http benchmarks have 350 reqs here

Reply:
Trying with 350, lets see the time that it needs now

Reply:
It looks like it is in time with 350 also, I can maintain it